### PR TITLE
Add page for Cypress.currentTest

### PR DESCRIPTION
**Pull request by jennifer-shehane**
Add page to explain what Cypress.currentTest does technically. For https://github.com/cypress-io/cypress/pull/16652

I’d love this to have better examples of *why* someone would use this. Could likely look at https://github.com/cypress-io/cypress/issues/2972 for examples.

### CODE REVIEW

File: content/api/cypress-api/currenttest.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: Cypress.currentTest
+---
+
+`Cypress.currentTest` returns the currently executing test instance with
+properties to access the title of the test.
+
```
Comment (cowboy) [inline content/api/cypress-api/currenttest.md:7]: Can we add a note to this effect?

```suggestion

<Alert type="warning">

Note that `Cypress.currentTest` may only be used inside tests and [test
hooks](/guides/core-concepts/writing-and-organizing-tests#Hooks),
and will be `null` outside of tests and test hooks.

</Alert>
```

File: content/api/cypress-api/currenttest.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+```javascript
+// returns object with title and titlePath properties
+Cypress.currentTest
+
+// returns title of current test
+Cypress.currentTest.title
+
+// returns array with current test title path
+Cypress.currentTest.titlePath
```
Comment (chrisbreiding) [inline content/api/cypress-api/currenttest.md:18]:
```suggestion
// an object with title and titlePath properties
Cypress.currentTest

// the title of the current test
Cypress.currentTest.title

// an array with the current test's title path
Cypress.currentTest.titlePath
```

I feel like using `returns` indicates they are functions instead properties.

File: content/api/cypress-api/currenttest.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+---
+title: Cypress.currentTest
+---
+
+`Cypress.currentTest` returns the currently executing test instance with
```
Comment (chrisbreiding) [inline content/api/cypress-api/currenttest.md:5]:
```suggestion
`Cypress.currentTest` is an object representing the currently executing test instance, with
```

**Review by chrisbreiding — CHANGES_REQUESTED** [review]

**Review by cowboy — CHANGES_REQUESTED** [review]
Looks good. One minor request.

File: content/api/cypress-api/currenttest.md (hunk 1 above)
Comment (jennifer-shehane) [reply content/api/cypress-api/currenttest.md:7]: Ah, great point to add!

File: content/api/cypress-api/currenttest.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+properties to access the title of the test.
+
+<Alert type="warning">
+
+Note that `Cypress.currentTest` may only be used inside tests and [test
+hooks](/guides/core-concepts/writing-and-organizing-tests#Hooks),
+and will be `null` outside of tests and test hooks.
+
+</Alert>
+## Syntax
```
Comment (cowboy) [inline content/api/cypress-api/currenttest.md:15]: Whoops, these need a linebreak between them

```suggestion
</Alert>

## Syntax
```

**Review by cowboy — CHANGES_REQUESTED** [review]

**Review by chrisbreiding — APPROVED** [review]

**Pull request merged**